### ISSUE TITLE
Hr dark theme uses black instead of white as fill, making it invisible on dakr backgrounds

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-web-and-design/vanilla-framework#4186]
![image](https://user-images.githubusercontent.com/2741678/145062387-be124d74-941b-4daf-890f-e4a36cbd5e48.png)

**Comment by bethcollins92** [comment]
@lyubomir-popov - Not too sure what you mean by this. Currently, if you're using the `hr` on a dark background you can add the ".is-dark" class to it to use the light line fill. Is this not the expected behaviour?

**Comment by lyubomir-popov** [comment]
You are right, I don't know why I was expecting it to be working without the class. Closing this.
